README.md

```diff
@@ -45,6 +45,24 @@ $ cekit build --overrides "{version: <version_default_is_latest>}" docker
 
 ### Running the Infinispan Operator
 
+#### Adding Annotations
+
+Annotations are labels that {ispn_operator} propagates to pods and services that it controls.
```

Comment:
This is not annotations from deployment point of view. We are using annotations in the Infinispan CR just for identifying list of the labels for each type: Pods and Services.